core/domain/feedback_services.py

```diff
@@ -218,6 +218,12 @@ def create_suggestion(exploration_id, author_id, exploration_version,
     _enqueue_suggestion_email_task(exploration_id, thread_id)
 
 
+def create_report(exploration_id, report_type):
```

Comment:
Don't think you need this separate function, just call enqueue_report_email_task directly from the controller.